assets/js/components/setup/setup-wrapper.js

```diff
@@ -96,77 +68,75 @@ class SetupWrapper extends Component {
 		delay( function() {
 			global.location.replace( redirectURL );
 		}, 500, 'later' );
+	}, [] );
```

Comment:
I think here we should pass `moduleToSetup` from above as callback dependency and then use it in the callback, instead of relying on the same global again, which may in theory cause inconsistent callbacks.